src/components/settings/UISettings.svelte

```diff
@@ -37,7 +37,7 @@
     options={Object.keys(TextDirection)}
     store={textDirection}
   />
-  {#if !isStandalone}
+  {#if $displayMode !== DisplayMode.POPOUT}
     <EnumOption
       name="Video side:"
       options={Object.keys(VideoSide)}
```

Comment:
Probably better to change to only show on `FULLPAGE`, video side doesn't do anything for embed mode anyway.